include/fmt/core.h

```diff
@@ -394,22 +394,22 @@ template <typename T, typename Char>
 struct is_named_arg<named_arg<T, Char>> : std::true_type {};
 
 enum type {
-  NONE, NAMED_ARG,
+  FMT_NONE, FMT_NAMED_ARG,
   // Integer types should go first,
-  INT, UINT, LONG_LONG, ULONG_LONG, BOOL, CHAR, LAST_INTEGER_TYPE = CHAR,
+  FMT_INT, FMT_UINT, FMT_LONG_LONG, FMT_ULONG_LONG, FMT_BOOL, FMT_CHAR, FMT_LAST_INTEGER_TYPE = FMT_CHAR,
   // followed by floating-point types.
-  DOUBLE, LONG_DOUBLE, LAST_NUMERIC_TYPE = LONG_DOUBLE,
-  CSTRING, STRING, POINTER, CUSTOM
+  FMT_DOUBLE, FMT_LONG_DOUBLE, FMT_LAST_NUMERIC_TYPE = FMT_LONG_DOUBLE,
+  FMT_CSTRING, FMT_STRING, FMT_POINTER, FMT_CUSTOM
```

Comment:
Adding `FMT_` prefix is no longer needed as the constants have been renamed in https://github.com/fmtlib/fmt/commit/8ed264fcd49de5716204e77c6293e1a7aef46906.